### ISSUE TITLE
existence of license enclosure about commercial use of UNZIP

### ISSUE THREAD

**Issue by pmqs** [pmqs/z1#291]
### Posted by armerd » Thu Sep 12, 2013 10:15 pm
I hear it about the redistribution in the business of the automatic defrosting formal file compressed by Info-ZIP.\
There is description of the following of the purport that it may approve unless the usual SFX banner is removed as follows as an exception condition.
What does a SFX banner mean?

>--------------------------\
>..that iInfos permitted without inclusion of this license, as long as the normal SFX\
>banner has not been removed from the binary or disabled.\
>\--------------------------

**Comment by pmqs** [comment]
### Posted by sms » Fri Sep 13, 2013 2:43 pm
The "SFX banner" is the line like:

    UnZipSFX 6.00 of 20 April 2009, by Info-ZIP ([http://www.info-zip.org](https://web.archive.org/web/20180203064532/http://www.info-zip.org/)).

which UnZipSFX normally displays when it is run. (Look for
`UnzipSFXBanner` in `unzip.c.`) The license ("LICENSE") does not permit
you to remove or disable that message.

The idea is that the person who is using our program should know that
he is using our program, and he should have enough information to let
him find our license, if he wants to see it.

**Comment by pmqs** [comment]
### Posted by armerd » Tue Sep 17, 2013 6:50 pm
SFX banner is contained in the message displayed on self-extraction EXE instruction execution time.\
(Portion enclosed with the red of attached file `unzipsfx.png`)\
Although it seems that the exception condition is fulfilled, how is it?

Attachments


![unzipsfx.png](/web/20180203064532im_/http://www.info-zip.org/phpBB3/download/file.php?id=88&sid=07b5829bdc63cfd31516d9cd9facae59)\
    unzipsfx.png (38.03 KiB) Viewed 1302 times

**Comment by pmqs** [comment]
### Posted by sms » Tue Sep 17, 2013 8:33 pm
I'm not sure that I understand your question.

If you want to distribute UnZip programs other than UnZipSFX (for
example, normal UnZip), then you would normally need to include the
license ("LICENSE") in your distribution to satisfy license condition
"2".

The UnZipSFX exception in the license allows you to distribute a
self-extracting archive without also distributing the license
("LICENSE"). However, to use this exception, you must not remove or
disable the "SFX banner". If you do not modify the code, then UnZipSFX
should display the "SFX banner", and that will satisfy the license.

In other words: For a self-extracting archive which uses UnZipSFX,
displaying the "SFX banner" is good enough. For _other_ programs, you
must provide the whole license ("LICENSE").

Note that "UnZipSFX 5.51 of 22 May 2004" is pretty old. Version 6.00
is the current released version, and probably has fewer bugs.

**Comment by pmqs** [comment]
### Posted by armerd » Tue Sep 17, 2013 9:05 pm
Thank you for the quick reply.\
My question was solved by the obtained contents.

**Comment by pmqs** [comment]
>[!NOTE]
The text in this issue was sourced from [here](https://web.archive.org/web/20180203064532/http://www.info-zip.org/phpBB3/viewtopic.php?f=4&t=415&p=2579&sid=199aec12efc9bb647bd82078c5d9db1d)
